### ISSUE TITLE
Docs: installation options

### ISSUE THREAD

**Issue by almereyda** [modoboa/modoboa-installer#38]
As it appears `modoboa-installer` will be the main entry point for future users, it is advisable to update its documentation with more granular detail about its internal workings.

A first step could be to actually explain the `installer.cfg` file instead of referring to its inline comments. The devmode is, for example, not immediately understandable. Is it maybe even equal to appending the `--debug` switch? This needs to be cleared out.

Additionally a sanity check of the file before continuing with the setup may help in saving some people from headaches in the future? At least I am interested in requirements for the different keys available.

**Comment by tonioo** [comment]
Yes, it's a good idea. What would be a good place for this documentation ?

**Comment by almereyda** [comment]
As I have always felt the modoboa documentation at http://modoboa.readthedocs.io a little confusing, I can imagine to contribute to a documentation community initiative. For this to be successful, I would regard two steps mandatory for distribution of responsability:

* Fork `tonioo/modoboa` to `modoboa/modoboa` and
* Extract the `doc` subfolder into its own repository.

Then the commit trees of both projects would be separated and could be maintained independently.
Next up could then be a rewrite of [the Getting Started section](https://github.com/tonioo/modoboa/tree/master/doc/getting_started) to increase separation of the different deployment modes: 

* manually
* via the installer
* elsewhat - didn't modoboa even start out as a set of Ansible roles?).
